course/page/base.py

```diff
@@ -139,15 +139,57 @@ def markup_to_html(
 
 # {{{ answer feedback type
 
+
+class InvalidFeedbackPointsError(ValueError):
+    pass
+
+
+CORRECTNESS_ATOL = 1e-5
+
+
+def get_close_value(correctness):
+    # type: (Optional[float]) -> (Optional[Union[float, int]])
+
+    if correctness is None:
+        return None
+    elif abs(correctness - 0) < CORRECTNESS_ATOL:
```

Comment:
Systematically, we could maybe do this for every integer in the range up to `MAX_CREDIT_FACTOR`.

Reply:
Maybe for halves and quarters, too--since they are exactly representable in FP.

Reply:
That's fair.